packages/ppx/src/Property_to_runtime.re

```diff
@@ -2442,19 +2442,27 @@ let hanging_punctuation =
 
 let render_generic_family = (~loc) =>
   fun
-  | `Cursive => [%expr "cursive"]
-  | `Fantasy => [%expr "fantasy"]
-  | `Monospace => [%expr "monospace"]
-  | `Sans_serif => [%expr "sans-serif"]
-  | `Serif => [%expr "serif"]
-  | `_apple_system => [%expr "-apple-system"];
+  | `Cursive => [%expr `cursive]
+  | `Fantasy => [%expr `fantasy]
+  | `Monospace => [%expr `monospace]
+  | `Sans_serif => [%expr `sans_serif]
+  | `Serif => [%expr `serif]
+  | `System_ui => [%expr `system_ui]
+  | `Ui_serif => [%expr `ui_serif]
+  | `Ui_sans_serif => [%expr `ui_sans_serif]
+  | `Ui_monospace => [%expr `ui_monospace]
+  | `Ui_rounded => [%expr `ui_rounded]
+  | `Emoji => [%expr `emoji]
+  | `Math => [%expr `math]
+  | `Fangsong => [%expr `fangsong]
+  | `_apple_system => [%expr `custom("-apple-system")];
```

Comment:
apple-system isn't custom isn't it?

I believe is the old `system-ui`, but it shouldn't have the quotes

Reply:
you are right, I thought it was a font-name